scss/bitstyles/components/button-groups.stories.mdx

```diff
@@ -88,29 +87,97 @@ For a series of buttons that are very directly related e.g.
       </ul>
     `}
   </Story>
+  <Story name="Joined buttons - icons 2">
+    {`
+      <ul class="o-list-reset l-flex l-flex--inline l-items-stretch o-button--grouped-container">
+        <li class="l-flex">
+          <button class="o-button o-button--icon o-button--grouped o-button--grouped__first" type="button">
+            <svg class="o-button__icon o-icon o-icon--m" viewBox="0 0 100 100" width="20" height="20" xmlns="http://www.w3.org/2000/svg" role="img" aria-hidden="true">
+              <path d="M67.17,2.78a6.07,6.07,0,0,1,8.59,8.58h0L41.41,45.71a6.07,6.07,0,0,0,0,8.58L75.76,88.64a6.07,6.07,0,1,1-8.59,8.58L24.24,54.29a6.07,6.07,0,0,1,0-8.58Z" fill-rule="evenodd"/>
+            </svg>
```

Comment:
Caveat: I'm not sure what these files mean, or how to understand these stories. And who would read the tests

Would it make sense to extract these icons-strings into named variables? Or are the SVG-structures significant for what you want to show here?

```mdx
const leftCaretIcon = `<svg …</svg>`

…

<button class="o-button o-button--icon o-button--grouped o-button--grouped__first" type="button">
  ${leftCaretIcon}
</button>
```

Reply:
Yup that’s a great idea! I’ll try it out 👍

Reply:
@lwassermann some context: these are the start of a component library that our devs building admin sections for our products can use. Right now that’s just copy/paste into a project, in future that may mean we create Rails & Elixir components from them, that can be imported and used (and be more maintainable).

So these stories are not for tests, but to give a series of examples that can be copied. The documentation is at very early stages, and will definitely need improvements! If you have any suggestions, they’d be very welcome :D🙏

Reply:
Sadly right now I can’t get your suggestion about extracting icons working :/ It may mean changing the format of the stories to use a `Template()` function instead of straight HTML — I’ll add a ticket to look into that, because it would certainly make these more readable and maintainable

Reply:
https://github.com/bitcrowd/bitstyles/issues/386